subxt/src/backend/rpc/rpc_client_t.rs

```diff
@@ -101,3 +101,94 @@ impl<T: RpcClientT> RpcClientT for Box<T> {
         (**self).subscribe_raw(sub, params, unsub)
     }
 }
+
+#[cfg(feature = "reconnecting-rpc-client")]
+impl RpcClientT for reconnecting_jsonrpsee_ws_client::Client {
+    fn request_raw<'a>(
+        &'a self,
+        method: &'a str,
+        params: Option<Box<RawValue>>,
+    ) -> RawRpcFuture<'a, Box<RawValue>> {
+        use futures::FutureExt;
+
+        async {
+            self.request_raw(method.to_string(), params)
+                .await
+                .map_err(|e| RpcError::ClientError(Box::new(e)))
+        }
+        .boxed()
+    }
+
+    fn subscribe_raw<'a>(
+        &'a self,
+        sub: &'a str,
+        params: Option<Box<RawValue>>,
+        unsub: &'a str,
+    ) -> RawRpcFuture<'a, RawRpcSubscription> {
+        use futures::{FutureExt, StreamExt, TryStreamExt};
+        use reconnecting_jsonrpsee_ws_client::{RpcError as ReconnError, SubscriptionId};
+
+        // Helper for more human friendly error messages.
+        // Workaround for: https://github.com/paritytech/jsonrpsee/issues/1280
+        fn rpc_error(err: ReconnError) -> String {
```

Comment:
I'd probably just `.to_string()` the whole error and put up with the more verbose message for now to save all of the code, and then when that linked issure is resolved it'll just get a bit nicer :)